x-pack/test/spaces_api_integration/spaces_only/apis/index.ts

```diff
@@ -19,9 +19,6 @@ export default function spacesOnlyTestSuite({ loadTestFile }: FtrProviderContext
     loadTestFile(require.resolve('./get'));
     loadTestFile(require.resolve('./update'));
     loadTestFile(require.resolve('./update_objects_spaces'));
-    // FLAKY: https://github.com/elastic/kibana/issues/158711, https://github.com/elastic/kibana/issues/158366
-    // Also, https://github.com/elastic/kibana/issues/158918
-    // esArchiver fails with no_shard_available_action_exception after deleting indexes
-    // loadTestFile(require.resolve('./disable_legacy_url_aliases'));
+    loadTestFile(require.resolve('./disable_legacy_url_aliases'));
```

Comment:
Looks like this test was flaky from before the split, maybe we should enable the suits mentioned in the issue to test that this works:

> Afaik the easiest way to spot the problem is to unskip both https://github.com/elastic/kibana/blob/main/x-pack/test/saved_object_api_integration/spaces_only/apis/resolve_import_errors.ts#L134 and https://github.com/elastic/kibana/blob/main/x-pack/test/saved_object_api_integration/spaces_only/apis/update.ts#L56 and run flaky-test-runner, usually 100 runs is enough

Reply:
Triggered 100 runs: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/2376